lib/ssh_scan/database/mongo_db.rb

```diff
@@ -0,0 +1,40 @@
+require 'mongo'
+
+Mongo::Logger.logger.level = ::Logger::FATAL
+
+module   SSHScan
+  module Database
+    class MongoDb
+      def initialize(opts = {})
+        name = opts["database"]["name"]
```

Comment:
We're making a lot of assumptions here about what might be in the config file.  I wonder if we should have some error handling here to make sure we have a minmum set of information to work with to avoid runtime errors.  Thoughts?